Dockerfile

```diff
@@ -276,6 +276,10 @@ RUN pip install --upgrade cython && \
     pip install google-cloud-language==2.* && \
     pip install google-cloud-videointelligence==2.* && \
     pip install google-cloud-vision==2.* && \
+    # After launch install main
```

Comment:
Not sure I understand what this comment mean.

Reply:
I'll reword